src/screens/TopicPageScreen/MemoizedListComponent.js

```diff
@@ -154,6 +158,9 @@ const RenderListFeed = (props) => {
           hideThreeDot={true}
           props={item}
           height={getHeightHeader()}
+          isSelf={item?.is_self}
+          isFollow={item?.is_following_target}
+          onPressFollUnFoll={() => followUnfollowTopic(item, route?.params?.id, offset)}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `onPressFollUnFoll` function now calls `followUnfollowTopic` with parameters `item`, `route?.params?.id`, and `offset`. Ensure that these parameters are correctly validated before use to prevent potential issues.

223:
The new prop `isShowDM` is added but it's not clear from the provided context how it's being used. Ensure that it's properly validated and handled in the component.


<!-- This is an auto-generated comment by OSS CodeRabbit -->